mup.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
```

Comment:
you shouldn't probably commit keys, specific data to the repo .. 
you can move these keys to the `settings.json` files and we already ignore this file